changelog_entry.yaml

```
@@ -0,0 +1,5 @@
+- bump: minor
```

bug fix right? change `added:` to `fixed:` if so
```suggestion
- bump: patch
```